dao/mongo.go

```diff
@@ -55,7 +55,12 @@ func (m *Mongo) GetPaymentResource(id string) (*models.PaymentResource, error) {
 	}
 	defer paymentSession.Close()
 
-	c := paymentSession.DB("payments").C("payments")
+	cfg, err := config.Get()
```

Comment:
Could this refactoring of how the database is called also be implemented on line 43?

Reply:
Done